tests/features/joinup_search/group_search.feature

```diff
@@ -221,7 +221,7 @@ Feature: Search inside groups
     When I enter "ground" in the search bar and press enter
     Then the option with text "Chalet construction   (2)" from select facet "collection/solution" is selected
     And the page should show the tiles "Ground plan, Natural materials"
-    And the page should show the following chip:
+    And the page should show the following chip in the "Search bar":
```

Comment:
The errors were caused because there was styling applied in the ticket. The chip of the group now appeared twice in the page, one in the search bar, as it is meant, and one in the new chips. Probably one should be removed, however, since the tests are asserting for the chips in the search bar, I made it a bit more specific.